Sources/TerminalAPIKit/Models/Requests/InputRequest.swift

```diff
@@ -23,15 +23,22 @@ public final class InputRequest: Request {
     /// Information related to an Input request.
     public let inputData: InputData
     
+    /// Information to display and the way to process the display.
+    public let displayOutput: DisplayOutput?
+    
     /// Initializes the InputRequest.
     ///
     /// - Parameter inputData: Information related to an Input request.
-    public init(inputData: InputData) {
+    /// - Parameter displayOutput: Information to display and the way to process the display.
+
+    public init(inputData: InputData, displayOutput: DisplayOutput?) {
```

Comment:
nit - slightly 'nicer' to have a default value you think, i.e. `displayOutput: DisplayOutput? = nil`? will leave it to you